docs/source/API/containers/Unordered-Map.rst

```diff
@@ -114,8 +116,24 @@ Description
 
       Index where the key exists in the map as long as failed() == false
 
-Insertion
----------
+.. cppkokkos:struct:: template <class ValueTypeView, class ValuesIdxType> UnorderedMapInsertOpTypes
+
+   :tparam ValueTypeView: The UnorderedMap value array type.
+
+   :tparam ValuesIdxType: The index type for lookups in the value array.
+
+   .. rubric:: Public Methods
+
+   The first key inserted stores the associated value (default op).
```

Comment:
does this sentence belong here or should go somewhere else?

Reply:
I dont quite get why this: The first key inserted stores the associated value (default op). is here?

Reply:
Yes. That's the default behavior of the unordered map, the first k,v pair to 'win' the insertion is what sticks. All subsequent insertions will be discarded. How can we word this more clearly?

Reply:
Actually, this isn't new behavior. Let me check for existing wording.

Reply:
We currently state: `Insert the given key/value pair into the map`